OrganizelyAPI/OrganizelyAPI/Controllers/QuotesController.cs

```diff
@@ -105,21 +108,21 @@ public async Task<IActionResult> PutQuotes(int id, QuotesDTO quotesDTO)
 
         // POST: api/Quotes
         [HttpPost]
-        public async Task<ActionResult<QuotesDTO>> PostQuotes(QuotesDTO quotesDTO)
+        public async Task<ActionResult<Quotes>> PostQuotes([FromBody] QuotesDTO quotesDTO)
```

Comment:
QuotesDTO

Reply:
Yeah, we replaced QuotesDTO with Quotes to follow a similar format to the Course controller. Should we change it back?